docs/tenancy/1.x/tutorial-non-tenant.md

```diff
@@ -0,0 +1,161 @@
+
+## Routes
+The first thing we want to do is configure the [Routes Affect](affacts-routes).
+
+First start by creating a new file for all tenant specific routes.
+
+### Tenant Route File
+For this example it will be stored at `routes\tenant.php`
+```php
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| Tenant Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your tenant application.
+| They are not loaded by default, and will be loaded thought the Routes Affect.
+|
+*/
+
+Route::get('/', function () {
+    //This is the "root" route for all tenants
+});
+
+```
+
+### Routes Affect Listener
+Next we need to create the listener for the [Routes Affect]() that will get rid of the system (non-tenant) routes and load our tenant route file instead.
+
+`app\Listeners\ConfigureTenantRoutes.php`
+```php
+namespace App\Listeners;
+
+use Tenancy\Affects\Routes\Events\ConfigureRoutes;
+
+class TenantRoutes 
+{
+    public function handle(ConfigureRoutes $event) 
+    {
+        if($event->event->tenant)
+        {
+            $event
+                ->flush()
+                ->fromFile(
+                    ['middleware' => ['web']],
+                    base_path('/routes/tenant.php')
+                );
+        }
+    }
+}
+```
+
+#### Registering the listener
+Now update your `app\Providers\EventServiceProvider.php` to include your new listener.
+
+```php
+  protected $listen = [
+
+    ...
+
+    Tenancy\Affects\Routes\Events\ConfigureRoutes::class => [
+      App\Listeners\ConfigureTenantRoutes::class,
+    ]
+    
+    ...
+  ];
+```
+
+## Status
+
+At this point you application is configured to load any tenant routes when a tenant is identified. When a tenant is not identified, the routes located in `routes/web.php` will be used instead.
+
+## Tips and Reminders
+### Namespaceing
+As with any Laravel application; if you are storing controllers in a different location than in `app\Http\Controllers` you will need to use the namespace option within your routes.
+
+`routes\tenant.php`
+```php
+Route::namespace('Tenant')->group(function () {
+    // Controllers Within The "App\Http\Controllers\Tenant" Namespace
+});
+```
+
+### Sub-site by domains
+When using subdomains to identify your non-tenant sub-sites you will need to use the route option within your routes file.
+`routes\web.php`
+```php
+Route::domain('admin.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the admin site
+    });
+});
+
+Route::domain('www.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the public site
+    });
+});
+
+Route::get('/shared', function() {
+  //This is a route that is shared with both the admin site and the public site
+})
+```
+
+### Accessing `OnTenant` models from a "sub-site"
```

Comment:
Can we rename "sub-site" to tenant site or something else? And make it consistent in the whole tutorial.